cpp/src/phonenumbers/asyoutypeformatter.cc

```diff
@@ -89,31 +86,19 @@ PhoneMetadata CreateEmptyMetadata() {
 }  // namespace
 
 AsYouTypeFormatter::AsYouTypeFormatter(const string& region_code)
-    : regexp_factory_(new RegExpFactory()),
-      regexp_cache_(*regexp_factory_.get(), 64),
-      current_output_(),
-      formatting_template_(),
-      current_formatting_pattern_(),
-      accrued_input_(),
-      accrued_input_without_formatting_(),
-      able_to_format_(true),
-      input_has_formatting_(false),
-      is_complete_number_(false),
-      is_expecting_country_code_(false),
+    : regexp_factory_(new RegExpFactory()), regexp_cache_(*regexp_factory_, 64),
```

Comment:
I don't think that joining such lines as these here increase readability or maintainability, on the contrary, joining these lines will make any future changes to these lists require a reformatting. That's not an improvement.